pkg/acsengine/const.go

```diff
@@ -62,6 +62,7 @@ var KubeImages = map[api.OrchestratorVersion]map[string]string{
 		"dnsmasq":      "k8s-dns-dnsmasq-amd64:1.13.0",
 		"pause":        "pause-amd64:3.0",
 		"windowszip":   "v1.6.6intwinnat.zip",
+		"tiller":       "gcr.io/kubernetes-helm/tiller:v2.5.0",
```

Comment:
the full URL is constructed elsewhere, enabling China cloud.  I would just keep this as `tiller:v2.5.0`, and verify it constructs the url correctly